app/controllers/events_controller.rb

```diff
@@ -28,6 +28,22 @@ def create
     end
   end
 
+  def edit
```

Comment:
Because we're editing from the event show page, we don't need this action, actually. 😀